verity/src/helm/values.yaml

```diff
@@ -1,8 +1,13 @@
+## This will override the default app.kubernetes.io/instance label for all Objects.
+# instanceLabelOverride: verity
+
 deployment:
-  image: "177276529591.dkr.ecr.us-west-2.amazonaws.com/verity"
-  tag: "latest"
-  imagePullPolicy: "Always"
+  image: "609723053065.dkr.ecr.ap-southeast-1.amazonaws.com/verity"
+  tag: "0.0.1"
+  imagePullPolicy: "IfNotPresent"
```

Comment:
What if someone decide to use tag `latest` instead of any specific version? I think `imagePullPolicy: "Always"` is better here. Don't you think so?